### ISSUE TITLE
Save a .scss file in the FTP

### ISSUE THREAD

**Issue by kingbudda** [keegnotrub/coda-sass-plugin#29]
I'm totally new to scss and I installed the plugin in Coda. It works great when I am working locally. But when I save a scss file in the FTP directly on the server, no .css file is spawning.

So I wonder if this plugin automatically generates a css file locally and unable to automatically generate a css file directly on the server from a scss file on the server.

Thanks in advance!

**Comment by keegnotrub** [comment]
Yep, unfortunately the plugin only works locally for now.

See also #26 & #20

**Comment by danlanes** [comment]
I'm using the plugin and it's great.

Do you have any plans to release a version with the ability to generate a CSS file remotely?

**Comment by pixelkarma** [comment]
Yeah seriously, for those of use who use Coda for remote files (I'm guessing a huge percent), this awesome plugin is awesomely useless. Make it work remotely!

**Comment by exotec** [comment]
i aggree, it is only a great tool if it work with remote scss files. i hope the feature comes soon

**Comment by Rayken** [comment]
Still nothing?

**Comment by pixelkarma** [comment]
Seriously, still nothing?

**Comment by keegnotrub** [comment]
Not to be snarky, but I'm offering this plugin for free, and it is open-source to allow anyone to extend it. I'd love to have the time to spend looking into this issue, but it unfortunately doesn't keep the lights on for me.

If the general consensus is the plugin is awesomely useless I can pull this repo down as well as remove it from the Panic library in order to prevent future issues for people.

**Comment by guibaz** [comment]
Thanks Keegnotrub for giving your time and your plugin to the community, it's really appreciated! Indeed, it would have been great to have a remote scss feature but at least the plugin is working locally. :)
